cypress/e2e/createParkingLot.cy.js

```diff
@@ -0,0 +1,54 @@
+describe('Login Form', () => {
+    beforeEach(() => {
+        cy.cleanDB()
+        cy.populateWithDrivers()
+        cy.createAdmin()
+        cy.login('adminUsername', 'securePassword');
+        cy.visit('http://localhost:9000/create-parking-lot');
+    });
+  
+    it('successfully creates a parking lot', () => {
+      // --------
+      // ASSEMBLE
+      // --------
+  
+      // ---
+      // ACT
+      // ---
+      cy.get('[data-test-id="lot-name-input"]').type('Test lot 1');
+      cy.get('[data-test-id="lot-capacity-input"]').type("50");
+      cy.get('[data-test-id="create-parking-lot-submit-button"]').click();
+  
+      // ------
+      // ASSERT
+      // ------
+
+      //cy.get('[data-test-id=create-park-request-success-alert]').should('contain', 'Successfully created parking request');
```

Comment:
Why is this commented out? Are we not checking if the create operation succeeded?